docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+name: vita-min
+
+services:
+  db:
+    container_name: db
+    image: postgis/postgis:13-3.4-alpine
+    user: postgres
+    ports:
+      - 5432:5432
+    volumes:
+      - database:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+  jobs:
+    container_name: jobs
+    volumes:
+      - ./:/app
+    build:
+      dockerfile: Dockerfile.local
+      target: "base"
+    command: rails jobs:work
+    depends_on:
+      rails:
+        condition: service_healthy
+    environment:
+      - DOCKER=true
+      - RAILS_DB_HOST=db
+      - RAILS_DB_USERNAME=postgres
+      - RAILS_DB_PORT=5432
+      - RAILS_DB_PASSWORD=password
+  rails:
```

Comment:
Similarly to the above comment, we call the equivalent to this service `web` in our Procfiles - up to you if you feel that it's worth changing here for consistency.